### PR TITLE
eastus to centralindia

### PR DESCRIPTION
**Pull request by ram-mishra**
dharm request

### CODE REVIEW

**Comment by Newageoutlaw** [comment]
accepted

File: main.tf [hunk 1]
```diff
@@ -1,4 +1,4 @@
 resource "azurerm_resource_group" "akshay-rgs" {
   name     = "akshayrg"
-  location = "eastus"
+  location = "centralindia"
```
Comment (Newageoutlaw) [inline main.tf:3]: okay hai

**Review by Newageoutlaw — APPROVED** [review]
go ahead

File: main.tf (hunk 1 above)
Comment (ram-mishra) [reply main.tf:3]: ok

**Pull request merged**